sdk/textanalytics/ai-text-analytics/MIGRATION_v5_v6.md

```diff
@@ -0,0 +1,373 @@
+
+All of these methods can produce `AnalyzeResult`, `AnalyzeBatchResult[]`, and `AnalyzeBatchPoller` types respectively. Please see the section above about these types for more information.
+
+### Action specification
+
+In major version 5, there is one method per action for actions that are not long-running operations. For example, the `recognizeEntities` method perfoms entity recognition. In major version 6, there is only one method, `analyze`, that can perform one such action at a time. Compare the following samples using `recognizeEntities` and `analyze`:
+
+Previous (5.1.0):
+
+```typescript
+const client = new TextAnalyticsClient(...);
+const documents = [...];
+
+// Previously, there was one method per action for most actions
+const result = await client.recognizeEntities(documents);
+```
+
+Current (6.0.0-beta.1):
+
+```typescript
+const client = new TextAnalysisClient(...);
+const documents = [...];
+
+// Now, an action can be performed by passing its name
+const result = await client.analyze("EntityRecognition", documents);
+```
+
+On the other hand, the way to consume the results of such actions didn't change. Furthermore, in major version 5, long-running operations actions can be batched and performed using the `beginAnalyzeActions` method. In major version 6, the same can be done using the `beginAnalyzeBatch` method but with the following two major differences:
+
+- Healthcare analysis is now supported. In major version 5, healthcare analysis is performed exclusively using the `beginAnalyzeHealthcareEntities` method. In major version 6, it is supported exclusively as an action in `beginAnalyzeBatch`.
+- Each action and their result is identified by a `kind` field in a list. In major version 5, specifying an action list of entity recognition and entity linking is done as `{ recognizeEntities: [ { ... } ], recognizeLinkedEntities: [ { ... } ] }`. In major version 6, the same is done as `[ { kind: "EntityRecognition", ... }, { kind: "EntityLinking", ... } ]`.
+
+To look at those changes more closely, compare the following samples:
+
+Healthcare analysis:
+
+Previous (5.1.0):
+
+```typescript
+const client = new TextAnalyticsClient(...);
+const documents = [...];
+
+// Previously, healthcare analysis is done by calling the following method
+const poller = await client.beginAnalyzeHealthcareEntities(documents);
+```
+
+Current (6.0.0-beta.1):
+
+```typescript
+const client = new TextAnalysisClient(...);
+const documents = [...];
+
+// Now, it can be performed by passing it as a batch action
+const poller = await client.beginAnalyzeBatch([{ kind: "Healthcare" }], documents);
+```
+
+Specifying input actions:
+
+Previous (5.1.0):
+
+```typescript
+const client = new TextAnalyticsClient(...);
+const documents = [...];
+
+// Previously, healthcare analysis is done by calling the following method
```

Comment:
```suggestion
```

Reply:
Any reason in particular why to delete this?

Reply:
looks like copy/paste error. the example underneath doesn't show healthcare

Reply:
Oh good catch! I checked the first occurrence and didn't realize it was copied somewhere else. I updated the comment accordingly :)